Classes/Hooks/AbstractTranslateHook.php

```diff
@@ -74,6 +75,13 @@ protected function createTranslateContext(string $content, int $targetLanguageUi
 
         $targetLanguageRecord = $this->languageService->getTargetLanguage($site, $targetLanguageUid);
```

Comment:
IMO this should already be wrapped in a try-catch to catch "invalid" language determination, and having the check below (with null isntead of false) as contained.

Either adding parent exectpion to the chain or throwing new exception if return value is null.

Thinking about that - maybe we should not make the getTrgetLangauge() nullable return value now and instead introduc concrete exception for all cases.